### PR TITLE
Fixes cloaks on taurs again AGAIN

### PR DESCRIPTION
**Pull request by Heroman3003**
Every time it's fixed in one place, it breaks in another. I am lowkey growing to hate clip mask feature.

### CODE REVIEW

**Comment by KasparoVy** [comment]
What exactly is the bug? Is stuff not rendering, or being clipped when it shouldn't, etc?

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -781,7 +781,7 @@ var/global/list/damage_icon_parts = list() //see UpdateDamageIcon()
 	var/tail_is_rendered = (overlays_standing[TAIL_LAYER] || overlays_standing[TAIL_LAYER_ALT])
 	var/valid_clip_mask = (tail_style && tail_style.clip_mask_icon && tail_style.clip_mask_state)
 
-	if(!(istype(suit)) || (tail_is_rendered && valid_clip_mask && !(suit && suit.taurized))) //Clip the lower half of the suit off using the tail's clip mask for taurs since taur bodies aren't hidden.
+	if(tail_is_rendered && valid_clip_mask && !(suit && istype(suit) && suit.taurized)) //Clip the lower half of the suit off using the tail's clip mask for taurs since taur bodies aren't hidden.
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Makes more sense having it this way (putting it with the check to see if there's a suit at all), good thinking

**Comment by Heroman3003** [comment]
Bug changed. First it runtimed and didn't render on taurs. Then it just didn't render on taurs. Then it rendered on taurs but runtimed and didn't render if person lacked a tail. Hopefully now it renders always.

**Comment by KasparoVy** [comment]
Thanks much for the explanation, I appreciate it

**Pull request merged**